sqlalchemy_views/views.py

```diff
@@ -31,18 +31,8 @@ class CreateView(_CreateDropBase):
 
     __visit_name__ = "create_view"
 
-    def __init__(self, element, selectable, on=None, bind=None,
-                 or_replace=False, options=None):
-        try:
-            super(CreateView, self).__init__(element, on=on, bind=bind)
-        except TypeError:
-            # Since version 1.4.0 of SQLAlchemy the ** on ** parameter no
-            # longer exists. it causes a ** TypeError ** exception
-            if on is not None:
-                raise TypeError("'on' is not supported on SQLAlchemy 1.4+")
-
-            super(CreateView, self).__init__(element, bind=bind)
-
```

Comment:
Can we add additional logic here to accommodate a check for whether `bind` is supported?

It may be best at this point to do an explicit version comparison:

- If version is less than 1.4, we call `super(CreateView, self).__init__(element, on=on, bind=bind)`
- If version is >= 1.4 but < 2.0, we throw a TypeError if on is present, otherwise call `super(CreateView, self).__init__(element, bind=bind)`
- Otherwise, throw TypeError if either `on` or `bind` is set, then call `super(CreateView, self).__init__(element)`